docs-chef-io/content/workstation/getting_started.md

```diff
@@ -19,60 +19,37 @@ new Chef Workstation installation. If you do not have Chef Workstation
 installed, see its [installation guide](/workstation/install_workstation/)
 before proceeding further.
 
-## Configure Ruby Environment
+## Install a Code Editor
 
-For many users of Chef, the version of Ruby that is included in Chef
-Workstation should be configured as the default version of Ruby on your
-system.
-
-{{< note >}}
-
-These instructions are intended for macOS and Linux users.
-
-{{< /note >}}
-
-1. Open a terminal and enter the following:
-
-    ``` bash
-    which ruby
-    ```
-
-    which will return something like `/usr/bin/ruby`.
-
-2. To use Chef Workstation-provided Ruby as the default Ruby on your system, edit the `$PATH` and `GEM` environment variables to include paths to Chef Workstation. For example, on a machine that runs Bash, run:
-
-    ``` bash
-    echo 'eval "$(chef shell-init bash)"' >> ~/.bash_profile
-    ```
+A good visual code editor is not a requirement for working with Chef
+Infra, but a good code editor can save you time. A code editor should
+support the following: themes, plugins, snippets, syntax Ruby code
+coloring/highlighting, multiple cursors, a tree view of the entire
+folder/repository you are working with, and a Git integration.
 
-    where `bash` and `~/.bash_profile` represents the name of the shell.
+These are a few common editors:
 
-    If zsh is your preferred shell then run the following:
+- [Visual Studio Code (free/open source)](http://code.visualstudio.com)
+- [GitHub Atom - (free/open source)](http://atom.io)
```

Comment:
We probably want to remove mention of Atom at some point. The integration isn't maintained and Atom is slowly dying.